style.css

```diff
@@ -1,3 +1,16 @@
+
+h1 {
+    font-family: ‘Metrophobic’, Arial, serif; font-weight: 400;
```

Comment:
Please write multiple css rules on separate lines (i.e. move the font-weight to its own line)
Actually, isn't 400 the default? I think you can remove it all together